<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplePlugin</class>
 <widget class="QDockWidget" name="SamplePlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Doc&amp;kWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="_btn_home">
        <property name="text">
         <string>Home Position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_place">
        <property name="text">
         <string>Place Bottle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_sparse">
        <property name="text">
         <string>Sparse Stereo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_pose">
        <property name="text">
         <string>Pose Estimation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_sparse_test">
        <property name="text">
         <string>Sparse Stereo Test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_performTask">
        <property name="text">
         <string>Perform Pick and Place From Side</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_performTaskTop">
        <property name="text">
         <string>Perform Pick and Place From Top</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="_label">
        <property name="text">
         <string>Label</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
